.github/workflows/build_push.yml

```diff
@@ -93,6 +106,10 @@ jobs:
           sha=`sha256sum mihon-x86_64-${{ env.VERSION_TAG }}.apk | awk '{ print $1 }'`
           echo "APK_X86_64_SHA=$sha" >> $GITHUB_ENV
 
+          mv app/build/outputs/apk/foss/release/app-foss-universal-release-unsigned-signed.apk mihon-${{ env.VERSION_TAG }}.apk
```

Comment:
```suggestion
          mv app/build/outputs/apk/foss/release/app-foss-universal-release-unsigned-signed.apk mihon-foss-${{ env.VERSION_TAG }}.apk
```

Reply:
Btw, why do you use `mv` for universal build and `cp` for ABIs?